src/components/common/Tabs/TabItem.tsx

```diff
@@ -3,20 +3,13 @@ import styled from 'styled-components'
 
 interface TabProps {
   title: string
-  readonly id: number
-  onTabClick: (arg: number) => void
-  isActive: boolean
-  readonly activeColor?: string
+  onTabClick: () => void
+  readonly activeColor: string | false
 }
 
-interface WrapperProps {
-  readonly activeColor?: string
-  isActive: boolean
-}
-
-const Wrapper = styled.li<WrapperProps>`
-  background: ${({ isActive, activeColor }): string => `var(${isActive ? activeColor : '--color-primary'})`};
-  color: ${({ isActive }): string => `var(--color-text-${isActive ? 'primary' : 'secondary2'})`};
+const Wrapper = styled.li<Pick<TabProps, 'activeColor'>>`
```

Comment:
Here we use "Pick" to only grab certain types from TabProps?
`Pick<TabProps, 'activeColor'>`

Reply:
https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys

Reply:
if i changed only `TabProps` i could forget to change the Wrapper one too so this is safer

Reply:
I see, so that makes `Pick` superior here, given I don't have to have a 'duplicate' type interface.